ms-campus-kid-school/src/routes/sites_routes.go

```diff
@@ -1,19 +1,21 @@
-package main
+package routes
 
 import (
 	"encoding/json"
+	"ms_campus_kid_school/src/controllers"
+	"ms_campus_kid_school/src/utils"
 	"net/http"
 
 	"github.com/gorilla/mux"
 )
 
-func setupRoutesForSites(router *mux.Router) {
+func SetupRoutesForSites(router *mux.Router) {
 	// First enable CORS. If you don't need cors, comment the next line
 	enableCORS(router)
```

Comment:
Why is it necessary to execute this for all routes?

Reply:
it isn't for all routes, just need to be executed by the main, file that's in other package.